tests/namespace/test_utilities_namespace.py

```diff
@@ -0,0 +1,87 @@
+import importlib
+import pathlib
+
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+namespace_data = pathlib.Path(__file__).parent / 'namespace-utilities.txt'
+with open(namespace_data) as f:
+    namespace = f.read().splitlines()
+    # ignore commented data
+    namespace = [n.split(', ')[0] for n in namespace if not n.startswith('#')]
+
+
+@pytest.mark.parametrize('name', namespace)
+def test_utilities_namespace(name):
+    with pytest.warns(PyVistaDeprecationWarning):
+        import pyvista.utilities as utilities
```

Comment:
To make `black` happy:
```suggestion
        import pyvista.utilities as utilities

```

Reply:
May need to make flake8 happy with a `noqa: F401` too

Reply:
`pre-commit` is green. The import is not unused because we use it on the next line :)

Reply:
🤦🏻‍♂️ I wish it would show the following lines like it does the previous lines in these comment views on the "Conversation" page for this reason